AddinFiles/main.jsl

```diff
@@ -20,6 +19,8 @@ If(
 );
 ns:logusage( "Get PI Data" );
 
+include("queries.jsl");
```

Comment:
Could this be loaded in `addinload.jsl` around line 24 instead of here so the UI doesn't have to wait to read the file every time and just to keep all the loaded files together?  If there is a reason for it be here that is fine..